cli/src/bin/phylum.rs

```diff
@@ -174,12 +175,12 @@ async fn handle_commands() -> CommandResult {
         "version" => handle_version(&app_name, ver),
         "update" => handle_update(sub_matches).await,
         "parse" => handle_parse(sub_matches),
-        "ping" => handle_ping(api.await?).await,
-        "project" => handle_project(&mut api.await?, sub_matches).await,
-        "package" => handle_get_package(&mut api.await?, sub_matches).await,
-        "history" => handle_history(&mut api.await?, sub_matches).await,
-        "group" => handle_group(&mut api.await?, sub_matches).await,
-        "analyze" | "batch" => handle_submission(&mut api.await?, &matches).await,
+        "ping" => handle_ping(Spinner::wrap_with_message(api, "Pinging server...").await?).await,
```

Comment:
The message `"Pinging server..."` wraps the `api` future, but that future only includes reaching out to the OIDC provider. The "ping" to the actual API server is separate and won't be wrapped by this spinner.

Reply:
Yea, mostly just changed this line to demonstrate usage, I can revert it.